server/command.go

```diff
@@ -398,39 +398,43 @@ func parseTriggers(triggersCsv string) *gitlab.AddWebhookOptions {
 	}
 }
 
-// SubscribeCommand proccess the /gitlab subscribe command.
-// It returns a message and handles all errors my including helpful information in the message
-func (p *Plugin) subscribeCommand(parameters []string, channelID string, config *configuration, info *gitlab.GitlabUserInfo) string {
-	features := "merges,issues,tag"
+func (p *Plugin) subscriptionDelete(info *gitlab.GitlabUserInfo, config *configuration, fullPath, channelID string) (string, error) {
+	normalizedPath := normalizePath(fullPath, config.GitlabURL)
+	if deleted, err := p.Unsubscribe(channelID, normalizedPath); err != nil {
+		p.API.LogError("can't unsubscribe channel in command", "err", err.Error())
+		return "Encountered an error trying to unsubscribe. Please try again.", nil
+	} else if !deleted {
+		return "Subscription not found, please check repository name.", nil
+	}
```

Comment:
Nit pick: At mattermost the patter used is usually
```suggestion
	deleted, err := p.Unsubscribe(channelID, normalizedPath);
	if err != nil {
		p.API.LogError("can't unsubscribe channel in command", "err", err.Error())
		return "Encountered an error trying to unsubscribe. Please try again.", nil
	} 
	
	if !deleted {
		return "Subscription not found, please check repository name.", nil
	}
```